addons/bestja_estimation_reports/models.py

```diff
@@ -0,0 +1,194 @@
+                ('organization.coordinator', '=', self.env.uid),
+        ],
+        ondelete='restrict',
+    )
+    date = fields.Date(
+        required=True,
+        string="dzień:",
+    )
+    name = fields.Char(string="Nazwa projektu", related="project.name")
+    report_entries = fields.One2many(
+        'bestja.estimation_report_entry', 
+        inverse_name="estimation_report", 
+        string="sklep"
+    )
+    state = fields.Selection(STATES, default='draft', string="status:")
+    organization = fields.Many2one(
+        'organization',
+        string="Organizacja",
+        related='project.organization',
+    )
+    parent_project = fields.Many2one(
+        'bestja.project',
+        string="Projekt nadrzędny",
+        related='project.parent',
+        store=True,
+    )
+    responsible_organization = fields.Many2one(
+        'organization',
+        string="Zainteresowane organizacje",
+        compute="_compute_responsible_organization",
+        compute_sudo=True,
+        store=True,
+    )
+
+    _sql_constraints = [(
+        'estimation_report_unique',
+        'unique("project", "date")',
+        "Dla danego projektu i dnia może istnieć tylko jeden raport!"
+    )]
+
+    @api.one
+    @api.depends('parent_project', 'project')
+    def _compute_responsible_organization(self):
+        """
+        The organizations on the middle level (1) are responsible for managing
+        their reports and reports of their children.
+        This field can be used to group all reports managed by
+        a single organization together.
+        """
+        project = self.project
+        level = project.organization.level
+        if level == 1:
+            self.responsible_organization = project.parent.organization.id
+        elif level == 2:
+            self.responsible_organization = project.parent.parent.organization.id
+
+    @api.multi
+    def continue_action(self):
+        """
+        For the continue button, adds all stores related to the project and day.
+        """
+        domain = [('date', '=', self.date), ('store.project', '=', self.project.id)]
+        for day_in_store in self.env['bestja_stores.day'].search(domain):
+            self.env['bestja.estimation_report_entry'].create({
+                'estimation_report': self.id,
+                'day_in_store': day_in_store.id,
+                'tonnage': 0.0,
+                'store': day_in_store.store.id,
+                'store_project': day_in_store.store.project.id,
+            })
+
+    @api.one
+    def add_to_summary(self):
+        self.sudo().state = 'sent'
+
+    @api.model
+    def _needaction_domain_get(self):
+        """
+        Show sent count in menu.
+        """
+        return [
+            ('state', '=', 'sent'),
+        ]
+
+    @api.one
+    @api.constrains('date', 'project')
+    def _check_date_in_project(self):
+        if not (self.date <= self.project.date_stop and self.date >= self.project.date_start):
```

Comment:
You could also write this as:

`if not self.project.date_start <= self.date <= self.project.date_stop`